eax/tests/aes128eax.rs

```diff
@@ -0,0 +1,91 @@
+//! AES-128-GCM tests
+
+#[macro_use]
+extern crate hex_literal;
+
+#[macro_use]
+mod common;
+
+use self::common::TestVector;
+use eax::aead::{generic_array::GenericArray, Aead, NewAead, Payload};
+use eax::Aes128Eax;
+
+/// EAX test vectors
+///
+/// <https://web.cs.ucdavis.edu/~rogaway/papers/eax.pdf>
+const TEST_VECTORS: &[TestVector<[u8; 16]>] = &[
+    TestVector {
```

Comment:
I think it would be better to use the `aead::dev::new_test` macro, but I was a bit late. I will create a follow-up PR.